include/dem/dem_properties.h

```diff
@@ -37,28 +37,31 @@ namespace DEM
 
   enum PropertiesIndex : int
   {
-    type         = 0,
-    dp           = 1,
-    rho          = 2,
-    v_x          = 3,
-    v_y          = 4,
-    v_z          = 5,
-    acc_x        = 6,
-    acc_y        = 7,
-    acc_z        = 8,
-    force_x      = 9,
-    force_y      = 10,
-    force_z      = 11,
-    omega_x      = 12,
-    omega_y      = 13,
-    omega_z      = 14,
-    mass         = 15,
-    mom_inertia  = 16,
-    M_x          = 17,
-    M_y          = 18,
-    M_z          = 19,
-    displacement = 20,
-    n_properties = 21,
+    type             = 0,
+    dp               = 1,
+    rho              = 2,
+    v_x              = 3,
+    v_y              = 4,
+    v_z              = 5,
+    acc_x            = 6,
+    acc_y            = 7,
+    acc_z            = 8,
+    force_x          = 9,
+    force_y          = 10,
+    force_z          = 11,
+    omega_x          = 12,
+    omega_y          = 13,
+    omega_z          = 14,
+    mass             = 15,
+    mom_inertia      = 16,
+    M_x              = 17,
+    M_y              = 18,
+    M_z              = 19,
+    displacement     = 20,
+    acc_derivative_x = 21,
+    acc_derivative_y = 22,
+    acc_derivative_z = 23,
```

Comment:
This is used to store a finite difference of the acceleration I guess? It would be good to have some sort of comments or something here to explain what this is.